proxy/proxy.go

```diff
@@ -966,8 +966,7 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 				status = http.StatusServiceUnavailable
 			}
 			p.tracing.setTag(ctx.proxySpan, HTTPStatusCodeTag, uint16(status))
-			//lint:ignore SA1019 Temporary is deprecated in Go 1.18, but keep it for now (https://github.com/zalando/skipper/issues/1992)
-			return nil, &proxyError{err: fmt.Errorf("net.Error during backend roundtrip to %s: timeout=%v temporary='%v': %w", req.URL.Host, nerr.Timeout(), nerr.Temporary(), err), code: status}
+			return nil, &proxyError{err: fmt.Errorf("%T during backend roundtrip to %s: timeout=%v: %w", nerr, req.URL.Host, nerr.Timeout(), err), code: status}
```

Comment:
Would `%T` show the interface? Not that we need a fixed `net.Error` here but this log message will of course change.

Reply:
No it won't. It'll just show `net.Error` as type.

Reply:
ok